.github/CHANGELOG.md

```diff
@@ -81,75 +81,74 @@
 
   1. Configuring account credentials using:
 
-    * Strawberry Fields v0.19.0
+     * Strawberry Fields v0.19.0
```

Comment:
Why the added space? Isn't 4 spaces good? 🤔

Reply:
Nope! It turns out we needed one more space for the changelog to render as intended.